.github/workflows/google-fonts.yml

```
@@ -74,14 +73,20 @@ jobs:
       - name: Create commits
         if: env.changes == 'true'
         run: |
-          git add .
-          git commit \
-            --message "${{matrix.mode}} Google Fonts" \
-            --message "This commit was created automatically by the [\`google-fonts\`](https://github.com/Homebrew/homebrew-cask/blob/master/.github/workflows/google-fonts.yml) workflow."
+            git ls-files --modified --others -z | xargs -0 -I {} sh -c "
+              FILENAME=\"{}\"
+              MESSAGE=\$(basename \"\$FILENAME\" .rb)
+              git add \"\$FILENAME\"
+              git commit \"\$FILENAME\" --message=\"\$MESSAGE: \$MODE\" --message=\"\$COMMIT_BODY\""
```

```suggestion
            git ls-files --modified --others -z | xargs -0 -I {} sh -c '
              MESSAGE="$(basename "{}" .rb)"
              git add "{}"
              git commit "{}" --message="$MESSAGE: $MODE" --message="$COMMIT_BODY"'
```
You probably don't need all the escaping. If `MODE` and `COMMIT_BODY` are missing then try doing `export MODE COMMIT_BODY` before calling `xargs`